ml-agents/mlagents/trainers/learn.py

```diff
@@ -153,7 +153,7 @@ def _create_parser():
     )
     argparser.add_argument(
         "--base-port",
-        default=UnityEnvironment.BASE_ENVIRONMENT_PORT,
+        default=5005,
```

Comment:
I think I'm going to echo that we should leave this public and read it from there. So that we can change the base port in one place.

Reply:
It actually does not matter if this `base-port` and `BASE_ENVIRONMENT_PORT` are different. I see `BASE_ENVIRONMENT_PORT` as a fallback, not as a default.